inference-engine/src/mkldnn_plugin/nodes/mkldnn_deconv_node.h

```diff
@@ -56,8 +57,8 @@ class MKLDNNDeconvolutionNode : public MKLDNNNode {
         protected:
             class IntermReorder {
                 public:
-                    static IntermReorder createFromReorder(MKLDNNMemoryPtr mem, const mkldnn::memory::desc& desc, const mkldnn::engine& engine);
-                    static IntermReorder createToReorder(MKLDNNMemoryPtr mem, const mkldnn::memory::desc& desc, const mkldnn::engine& engine);
+                    IntermReorder(MKLDNNMemoryPtr mem, const mkldnn::memory::desc& desc, const mkldnn::engine& engine);
```

Comment:
Suggest changing names `mem` -> `memFrom` and `desc` -> `descTo` and vice versa for the second constructor, so the names reflect its purpose.